package.json

```diff
@@ -31,42 +31,47 @@
   },
   "license": "Apache-2.0",
   "devDependencies": {
-    "@babel/core": "^7.17.8",
-    "@babel/eslint-parser": "^7.18.2",
-    "@babel/preset-env": "^7.16.11",
+    "@babel/core": "^7.18.13",
+    "@babel/eslint-parser": "^7.22.10",
+    "@babel/preset-env": "^7.22.10",
     "@duckduckgo/content-scope-scripts": "github:duckduckgo/content-scope-scripts#1.3.0",
     "@duckduckgo/content-scope-utils": "github:duckduckgo/content-scope-utils#1.0.1",
-    "@playwright/test": "^1.32.0",
-    "@types/jest": "^27.4.1",
-    "@types/node": "^16.11.36",
+    "@playwright/test": "^1.37.0",
+    "@types/jest": "^29.5.3",
+    "@types/node": "^18.17.5",
     "asana": "^1.0.0",
-    "babel-jest": "^27.5.1",
+    "babel-jest": "^29.6.2",
     "babelify": "^10.0.0",
-    "eslint": "^7.32.0",
+    "eslint": "^8.47.0",
     "eslint-config-standard": "11.0.0",
-    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-import": "^2.28.0",
     "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-promise": "^6.0.0",
+    "eslint-plugin-promise": "^6.1.1",
     "eslint-plugin-standard": "^5.0.0",
     "fast-check": "^2.25.0",
-    "grunt": "^1.5.2",
+    "grunt": "^1.6.1",
     "grunt-babel": "^8.0.0",
     "grunt-browserify": "^6.0.0",
     "grunt-contrib-watch": "^1.1.0",
-    "grunt-eslint": "^24.0.0",
+    "grunt-eslint": "^24.3.0",
     "grunt-exec": "^3.0.0",
-    "jest": "^27.5.1",
-    "jest-chrome": "^0.7.2",
-    "jest-html-reporter": "^3.4.2",
-    "json-schema-to-typescript": "^11.0.2",
+    "http-server": "^14.1.1",
+    "jest": "^29.6.2",
+    "jest-chrome": "^0.8.0",
+    "jest-environment-jsdom": "^29.6.2",
+    "jest-html-reporter": "^3.10.1",
+    "json-schema-to-typescript": "^13.0.2",
     "load-grunt-tasks": "^5.1.0",
     "markdown-it": "^13.0.1",
-    "playwright": "^1.31.2",
+    "playwright": "^1.37.0",
     "through2": "^4.0.2",
-    "ts-to-zod": "^1.13.1",
-    "typescript": "^4.7.2",
-    "zod": "^3.16.0",
-    "http-server": "^14.1.1"
+    "ts-to-zod": "^3.1.3",
+    "typescript": "^5.1.6",
+    "zod": "^3.21.4"
   },
-  "dependencies": {}
+  "overrides": {
+    "jest-chrome": {
+      "jest": "$jest"
+    }
+  }
```

Comment:
Needed because `jest-chrome` is set to be dependent to jest 27, but it's not actually necessary. They've already merged the necessary change but haven't released (it's been a few months), so this is temporary.